<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOCListWidgetItem</class>
 <widget class="QWidget" name="TOCListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>82</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>82</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>82</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #404040;
box-shadow: inset 0px -1px 1px rgba(0, 0, 0, 0.15);
margin: 0px 0px;

</string>
  </property>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>27</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/toc-reading-order.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Href">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>26</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	font-family: Noto Sans TC;
	font-style: normal;
	font-weight: normal;
	font-size: 15px;
	line-height: 20px;

	color: #FFFFFF;


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: #FFFFFF;

	font-family: Noto Sans TC;
	font-style: normal;
	font-weight: normal;
	font-size: 15px;
	line-height: 20px;
	background: #404040;
}

QLineEdit:focus {
	background: #393939;
}

QLineEdit:disabled
{
	border: 3px solid rgba(255, 0, 0, 1);
	background: rgba(0, 0, 0, 0.2);
	box-sizing: border-box;
	border-radius: 6px;
	color: #FFFFFF;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Type here...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Remove">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>12</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/common-remove.svg</normaloff>:/SVG/svg/icon/common-remove.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>53</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/toc-start-time.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_Start">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>48</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFFFF;

font-family: Noto Sans;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 19px;


background: rgba(0, 0, 0, 0.2);
border-radius: 6px;

padding-left: 2px;</string>
   </property>
   <property name="minimumTime">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string notr="true">hh:mm:ss.zzz</string>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>0</y>
     <width>300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #333333;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Drag">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>32</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/common-drag.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
